src/rational.rs

```diff
@@ -113,6 +130,14 @@ impl Rational {
     }
 
     /**
+     * Returns this Rational to the nearest Int
+     */
+    pub fn round(&self) -> Int {
+        let r = self + Rational::new(self.sign().into(), 2.into());
```

Comment:
This should probably be replaced with something like:

```rust
let mut r = self + self.sign();
r.d += 2;
```

Otherwise it has to allocate twice when constructing the `±1/2` rational.